sdk/keyvault/keyvault-admin/package.json

```diff
@@ -56,8 +56,8 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "generate:client": "autorest --typescript swagger/README.md",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "dev-tool run test:node-js-input --use-esm-workaround=true -- --timeout 180000 --exclude \"dist-esm/**/*.browser.spec.js\" \"dist-esm/**/*.spec.js\"",
-    "integration-test:node:no-timeout": "dev-tool run test:node-js-input --use-esm-workaround=true -- --no-timeouts --full-trace  --exclude \"dist-esm/**/*.browser.spec.js\" \"dist-esm/**/*.spec.js\"",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 180000 --exclude \"dist-esm/**/*.browser.spec.js\" \"dist-esm/**/*.spec.js\"",
```

Comment:
@mpodwysocki / @xirzec should these packages be switched to use `test:node-tsx-js` instead of `test:node-js-input` and if not why not? Many thanks!